src/dmochijson2.erl

```diff
@@ -461,17 +466,20 @@ tokenize_string(B, S=#decoder{offset=O}, Acc) ->
             end;
         <<_:O/binary, C1, _/binary>> when C1 < 128 ->
             tokenize_string(B, ?INC_CHAR(S, C1), [C1 | Acc]);
-        <<_:O/binary, C1, C2, _/binary>> when C1 >= 194, C1 =< 223,
-                C2 >= 128, C2 =< 191 ->
+        <<_:O/binary, C1, C2, _/binary>> when C1 >= 194, C1 =< 223, C2 >= 128, C2 =< 191 ->
             tokenize_string(B, ?ADV_COL(S, 2), [C2, C1 | Acc]);
-        <<_:O/binary, C1, C2, C3, _/binary>> when C1 >= 224, C1 =< 239,
-                C2 >= 128, C2 =< 191,
-                C3 >= 128, C3 =< 191 ->
+        <<_:O/binary, C1, C2, C3, _/binary>>
+            when C1 >= 224, C1 =< 239, C2 >= 128, C2 =< 191, C3 >= 128, C3 =< 191 ->
             tokenize_string(B, ?ADV_COL(S, 3), [C3, C2, C1 | Acc]);
-        <<_:O/binary, C1, C2, C3, C4, _/binary>> when C1 >= 240, C1 =< 244,
-                C2 >= 128, C2 =< 191,
```

Comment:
I won't lie this one looks better because it keeps the original intention (checking ranges) more readable.

Reply:
I _fixed_ it in an extremely dumb way… 🙈

Reply:
I think it's fine since we're talking about `dmochijson2`, a JSON-parsing module stolen from mochiweb… 🤦